sdk/storage/Azure.Storage.Blobs/src/BlobClient.cs

```diff
@@ -756,10 +747,14 @@ protected internal virtual BlobClient WithClientSideEncryptionOptionsCore(Client
 
         /// <summary>
         /// The <see cref="Upload(Stream, BlobHttpHeaders, Metadata, BlobRequestConditions, IProgress{long}, AccessTier?, StorageTransferOptions, CancellationToken)"/>
-        /// operation creates a new block blob or updates the content of an
-        /// existing block blob.  Updating an existing block blob overwrites
+        /// operation overwrites the contents of the blob, creating a new block
+        /// blob if none exists.  Overwriting an existing block blob overwrites
```

Comment:
We're using "overwrite" a lot. Can we say: 

"Overwriting an existing block blob replaces any existing metadata on the blob."